e2e/test/scenarios/custom-column/cc-typing-suggestion.cy.spec.js

```diff
@@ -50,17 +50,15 @@ describe("scenarios > question > custom column > typing suggestion", () => {
     cy.contains("length([Title])");
   });
 
-  it(
-    "should correctly insert function suggestion with the opening parenthesis",
-    { tags: "@flaky" },
-    () => {
-      addCustomColumn();
-      enterCustomColumnDetails({ formula: "LOW{enter}" });
-
-      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-      cy.contains("lower(");
-    },
-  );
+  it("should correctly insert function suggestion with the opening parenthesis", () => {
+    addCustomColumn();
+    enterCustomColumnDetails({ formula: "BET{enter}" });
+
+    cy.findByTestId("expression-editor-textfield").should(
+      "contain",
+      "between(",
+    );
```

Comment:
This part wasn't needed, but I used the opportunity to get rid of the linter warning.